Gemfile

```diff
@@ -87,6 +87,7 @@ group :development, :test do
   gem 'aws-sdk-cloudwatchlogs', require: false
   gem 'brakeman', require: false
   gem 'bullet', '>= 6.0.2'
+  gem 'capybara-webmock', git: 'https://github.com/hashrocket/capybara-webmock.git', ref: '63d790a0'
```

Comment:
The reason this points to git is because there are unpublished changes that we need.

Specifically: https://github.com/hashrocket/capybara-webmock/commit/f3d27b1573626c1e150893c1ade3246b9a272ede